main.js

```diff
@@ -0,0 +1,40 @@
+let inputBox = document.getElementById("inputValue");
+let noTasks = document.getElementById('noTasks');
+
+// Getting today's date
+  n =  new Date();
+  y = n.getFullYear();
+  m = n.getMonth() + 1;
+  d = n.getDate();
+  document.getElementById("currentDate").innerHTML = m + "/" + d + "/" + y; 
+
+  window.onload = inputBox.focus();  
+
+  addTodoBtn.onclick = function addTask() { 
+  let inputValue = document.getElementById("inputValue").value; 
+  let deadlineDate = document.getElementById("deadlineDate").value; 
+  if (inputValue === '') 
+  {
+    alert("This field cannot be empty!");
+  }
+  else
+  {
```

Comment:
pay attention to the indentation here, the `{` should be in front of the `else` like this
```suggestion
  else {
```